.goreleaser.yml

```
@@ -0,0 +1,56 @@
+before:
+  hooks:
+    - go mod tidy
+    - go generate ./...
+dist: bin
+builds:
+  - <<: &build_defaults
+      binary: flux_webui
```

I think this should be 

```suggestion
      binary: flux-webui
```

as `_` in binary names are usually frowned upon.